packages/components/src/atoms/PaneCloseIcon/PaneCloseIcon.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import * as S from "./PaneCloseIcon.styled";
+
+export type PaneCloseIconType = {
+  onClick: () => void;
+  containerId?: string;
+  containerStyle?: React.CSSProperties;
+  iconStyle?: React.CSSProperties;
+};
+
+const PaneCloseIcon: React.FC<PaneCloseIconType> = (props) => {
+  const { containerId = "pane-close", containerStyle = {}, iconStyle = {}, onClick } = props;
+
+  return (
+    <S.PaneCloseIconContainer id={containerId} style={{ ...containerStyle }} onClick={onClick}>
+      <S.DoubleLeftOutlined style={{ ...iconStyle }} />
+    </S.PaneCloseIconContainer>
+  );
+};
+
+export default PaneCloseIcon;
```

Comment:
Function vs const with React.FC is one discussion

Using named vs default exports is another. Named exports have a bit better tooling support and default exports can sometimes have a bit odd behaviour (for instance when you rename a component with your IDE).

@devcatalin @topliceanurazvan @monojack  tagging you both these two discussions so we can make a decision and be consistent for all other components.

Reply:
You have a point here, I agree that we should go for named exports instead of the default.